internal/support/vwatch/vwatch.bm

```diff
@@ -151,15 +151,15 @@ SUB vwatch (globalVariables AS _OFFSET, localVariables AS _OFFSET)
         EXIT SUB
     END IF
 
-    IF vwatch_linenumber = vw_lastLine AND vw_setNextLine = 0 THEN EXIT SUB
```

Comment:
Rather than a _FALSE value (hence the use of NOT, I assume), vw_setNextLine will range from 0 to the total lines of the user's program. NOT (some other value) wouldn't be the same as NOT -1 (= 0). This looks like it could cause issues.

Reply:
I stand corrected. It's `vwatch_goto` that needs to go higher. Been a while.